api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseEncounterTranslator.java

```diff
@@ -37,5 +42,28 @@ protected Coding mapLocationToClass(Location location) {
 		}
 		return coding;
 	}
-	
+
+	protected List<CodeableConcept> mapEncounterTypeClass(EncounterType openmrsEncounterType) {
+		Coding coding = new Coding();
+		coding.setCode(openmrsEncounterType.getUuid());
```

Comment:
Do you think it makes sense to include a dummy system here? It might be useful as a way to tag this as OMRS-specific data.